firedrake/mg/mesh.py

```diff
@@ -80,6 +85,7 @@ def __getitem__(self, idx):
 
 def MeshHierarchy(mesh, refinement_levels,
                   refinements_per_level=1,
+                  order=1,
```

Comment:
Please document this. And ideally switch the docstring to [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) at the same time.

Reply:
`degree`